src/events/api/serializers.py

```diff
@@ -43,3 +43,38 @@ class Meta:
             "speakers"
 
         ]
+
+
+class TutorialDetailSerializer(serializers.ModelSerializer):
+    speakers = serializers.SerializerMethodField()
+    date = serializers.DateTimeField(format="%Y-%m-%d %H:%M:%S")
+    begin_time = serializers.DateTimeField(format="%Y-%m-%d %H:%M:%S")
+    end_time = serializers.DateTimeField(format="%Y-%m-%d %H:%M:%S")
```

Comment:
TutorialProposal doesn't has these fields: date, begin_time, end_time
You should use ProposedTutorialEvent, and date will need to generate by begin_time

Reply:
I change this part from TutorialProposal to ProposedTutorialEvent.